src/components/Article.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, text, date }) => (
```

Comment:
Мне кажется так выглядит рациональнее.

```suggestion
function Article({
  title,
  date,
  text,
}) {...
```

Reply:
Ну Миша именно так показывал в видео, по этому решил делать так

Reply:
Старий синтаксис проти нового, тільки й всього